src/msg/async/ProtocolV2.h

```diff
@@ -77,7 +77,8 @@ class ProtocolV2 : public Protocol {
   uint64_t peer_required_features;
   std::shared_ptr<AuthSessionHandler> session_security;
 
-  uint64_t cookie;
+  uint64_t client_cookie;
+  uint64_t server_cookie;
```

Comment:
I wonder if less confusing names would be "my_cookie" and "peer_cookie", since the client/server names are ambiguous when both sides can connect to each other

Reply:
agreed, @cyx1231st reminded me the same.

Reply:
I agree